backend/app/api/api_v1/routers/lookups/__init__.py

```diff
@@ -0,0 +1,15 @@
+# flake8: noqa
+from .lookups1 import (
+    lookup_geographies,
```

Comment:
technically it's not necessary to have all these imports, but I had to import something. In the real implementation I would expect that we could import the request/response types?